common/src/main/java/winterwolfsv/cobblemon_quests/tasks/CobblemonTask.java

```diff
@@ -140,14 +150,16 @@ public Text getAltTitle() {
 
         boolean displayGender = !(gender.equals("choice_any") || gender.isEmpty());
         boolean displayType = !(pokemon_type.equals("choice_any") || pokemon_type.isEmpty());
+        boolean displayForm = !(form.equals("choice_any") || form.equals("normal") || form.isEmpty());
```

Comment:
Here I filter out the "normal" choice as it would be weird to say "Catch x1 normal Ponyta" for example but that doesn't allow the user to know right away whether they can capture any variant or only the normal variant